float32 x
float32 y
float32 z
---
int64 flag
float32 x_offset
float32 y_offset
float32 z_offset
